packages/reporting/src/index.tsx

```diff
@@ -2,10 +2,14 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/app';
 import * as serviceWorker from './serviceWorker';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
```

Comment:
`DateTimePicker` has graduated from material lab and into "material-x". This will cause reporting to fail when we merge https://github.com/open-rmf/rmf-web/pull/615.